server/routerlicious/packages/services-client/src/historian.ts

```diff
@@ -138,6 +138,10 @@ export class Historian implements IHistorian {
     public async createSummary(summary: IWholeSummaryPayload): Promise<IWriteSummaryResponse> {
         return this.restWrapper.post<IWriteSummaryResponse>(`/git/summaries`, summary, this.getQueryString());
     }
+    public async deleteSummary(softDelete: boolean): Promise<void> {
+        const headers = { "Soft-Delete": softDelete };
```

Comment:
Aren't we always soft deleting document? Why the header?

Reply:
@TriDucNgo What was the decision around the `Soft-Delete` header? I see that the comments were marked as resolved but I couldn't find the discussion/decision about why we want to keep that header